.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
+target
```

Comment:
They are Scala worksheets... We make use a lot of this feature.
On Feb 4, 2016 2:18 PM, "mcovarr" <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/broadinstitute/cromwell-backend/pull/4#discussion_r51947153>
> :
>
> > @@ -0,0 +1,2 @@
> > +.idea
> > +target
>
> What are *.sc files?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/broadinstitute/cromwell-backend/pull/4/files#r51947153>
> .
>